codegen/test/rtc/src/compile_kernel.cpp

```diff
@@ -1,10 +1,16 @@
-#include "rtc/hip.hpp"
+#include <ck/host/stringutils.hpp>
```

Comment:
Dont include headers files from the host, just move the functions you need to this file.